### PR TITLE
bump Nan version number and add node 8 to the travis config

### PR DESCRIPTION
**Pull request by karadaisy**
Hello @silverlyra, 

Please review the following commits I made in branch 'kylewm/bump-version':

- **bump Nan version number and add node 8 to the travis config** (28496322e177cb558a6d0d91e7f7ef5109922590)

R=@silverlyra

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,36 +1,46 @@
...
-    "maintainers": [ {
-        "name": "Dan Bornstein",
-        "email": "danfuzz@obvious.com",
-        "web": "http://www.milk.com/"
-    } ],
-
+    "maintainers": [
+        {
+            "name": "Dan Bornstein",
+            "email": "danfuzz@obvious.com",
```
Comment (silverlyra) [inline package.json]: rofl

**Review by silverlyra — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (kylewm) [reply package.json]: Do you know about this dude? Apparently he wrote the Dalvik VM

Comment (kylewm) [reply package.json]: 🎶 I forgot these comments were public 🎶

Comment (danfuzz) [reply package.json:33]: 👋hi! 👋 FWIW I haven't had an `obvious.com` email address in a long time. You can copy my personal address from the block above here if you feel like it.

Comment (karadaisy) [reply package.json:33]: LOL hi Dan!

**Pull request merged**